Caltech-IPAC/firefly
# Firefly-1465: hips search Panel

#### Multiplie tickets
-  [Firefly-1465](https://jira.ipac.caltech.edu/browse/FIREFLY-1465): hips search Panel 
    - Includes api support for adding a new HiPS list server
- [Firefly-1429](https://jira.ipac.caltech.edu/browse/FIREFLY-1429): full descriptions of schemas and tables in TAP query screen via tooltip
- Fixed: expanded mode div in api sometime maintains size

#### Testing
- https://fireflydev.ipac.caltech.edu/firefly-1465-hips-panel/firefly
    - try HiPS Search Panel
    - url api if too hard to test here
- TAP: tooltips have full table description
- https://fireflydev.ipac.caltech.edu/firefly-1465-hips-panel/firefly/test/tests-main.html?test=IRSA+Gator+Table
   - in DevTools search dom for `expandedArea'
   - div should be empty when not expanded
